vtex-apps/return-app
# Calculate proportional shipping value

What this PR changes:
Add the option to calculate the delivery value proportionally to the total cost of the returned product(s) in the return request.

How to test it:
Toggle the Shipping value proportional to the total of the products to test the new feature here => [this workspace](https://zekiqa--vtexspain.myvtex.com/admin/returns/settings)
Then go to https://zekiqa--vtexspain.myvtex.com/account#/my-returns and create a return request. When finished go to details and you will see that the shipping value is proportional to the value of the products and the total of the order.

Here is an example with multiple products: https://zekiqa--vtexspain.myvtex.com/admin/returns/c7fe72a1-af9b-11ec-835d-123072a6b6a3/details/
And this one with more than one of the same product: https://zekiqa--vtexspain.myvtex.com/admin/returns/f12f40dc-aed7-11ec-835d-0aaa62e90a93/details/
